app/v1/controllers/DeleteBenefitAmountsController.scala

```diff
@@ -99,4 +117,11 @@ class DeleteBenefitAmountsController @Inject()(val authService: EnrolmentsAuthSe
       "SERVER_ERROR" -> DownstreamError,
       "SERVICE_UNAVAILABLE" -> DownstreamError
     )
+
+  private def auditSubmission(details: GenericAuditDetail)
+                             (implicit hc: HeaderCarrier,
+                              ec: ExecutionContext): Future[AuditResult] = {
+    val event = AuditEvent("DeleteStateBenefitAmount", "delete-state-benefit-amounts", details)
```

Comment:
I think this is named wrong in the Spec and should be `DeleteStateBenefitAmounts` with an **s**. Like `AmendStateBenefitAmounts` is